### ISSUE TITLE
`newtype`s are formatted on two lines with `break_single_constructors: false`

### ISSUE THREAD

**Issue by imuli** [jaspervdj/stylish-haskell#342]
As `newtype`s always have a single constructor, you would think that they
would obey `break_single_constructors`. However, with that set to false,
```haskell
newtype Foo = Foo Int
```
becomes
```haskell
newtype Foo
   = Foo Int
```

while `data` with a single constructor stay on one line.
```haskell
data Foo = Foo Int
```

**Comment by imuli** [comment]
Okay, having now found the place in the source that explicitly checks this _and_ the test case that claims the current behavior is correct, it seems like @jaspervdj wants this behavior. I still find it unintuitive - if it's not too much configuration bloat could we add a `break_newtypes` flag?

**Comment by EncodePanda** [comment]
I think this accident @imuli . newtype and data has to be consistent. let's fix that.

**Comment by stevladimir** [comment]
Hello, any progress on that? Maybe I can help somehow?

**Comment by jaspervdj** [comment]
Yeah this is accidental.  I would welcome a PR that fixes this :-)

**Comment by stevladimir** [comment]
IMO this should be the default behavior, but It is somewhat breaking change (people might get big diffs) if we just directly alter the current behavior. If this is acceptable I would go this way as I see no strong reasons to distinguish `newtype` and `data` in this respect.

Otherwise, @imuli has a [fix](https://github.com/haskell/stylish-haskell/issues/342) which should be backwards compatible in cost of introducing a new option.

Which one would you prefer?

**Comment by imuli** [comment]
I think the primary issue with my fix as it stands is that it doesn't provide any tests?

I also would have no complaints about somone removing the option in favor of `break_single_constructors`.
